preconditions/README.md

```diff
@@ -0,0 +1,58 @@
+# IOTICS PRECONDITIONS
+
+In order to guarantee the full streaming functionality of IOTICS, we would need:
```

Comment:
In order to guarantee the functionality of IOTICS, we would need to ensure the following services, ports and protocols are supported. This will need to be verified in corporate environments where firewalls and internet proxies may affect successful operation.